lib/tasks/notifications/open-enrollment.js

```diff
@@ -0,0 +1,48 @@
+const moment = require('moment'),
+      Plan = require('../../models/plan'),
+      Employee = require('../../models/employee'),
+      isWorker        = process.env.CONTEXT === 'worker',
+      winston         = require('winston');
+
+let {
+  debug, error
+} = winston;
+
+const debugPfx = 'Task :: OPEN ENROLLMENTS ::';
```

Comment:
I see your all caps, I raise you a calm down

```suggestion
const debugPfx = 'Task :: Open Enrollments ::';
```